docs/deps.md

```diff
@@ -0,0 +1,96 @@
+# Operating system
+We build CodeCompass under Linux. It is recommended to use a 64-bit operating system.
+
+# Disk requirements
+It is recommended to have at least 15 GB of free space and at least 2GB RAM before you begin for convenient work.
```

Comment:
I measured the disk space in my PC (10GB). The memory recommendation comes from the old CodeCompass documentation.